Commander X16 Programmer's Reference Guide.md

```diff
@@ -695,6 +702,7 @@ Some notes:
 * The `IOBASE` call returns $9F60, the location of the first VIA controller.
 * The `SETTMO` call has been a no-op since the Commodore VIC-20, and has no function on the X16 either.
 * The `MEMTOP` call additionally returns the number of available RAM banks in the .A register.
+* The `LOAD` call requires A to be set to the secondary address instead of Load/Verify.
```

Comment:
BASIC's LOAD statement can't set the accumulator directly.  Therefore, a trick is used:  if its secondary address is greater than 1, that number is put into the accumulator.

But, this note must be for Assembly code, not BASIC.  It should say,
```
* When you want to load into VERA's RAM, set the .A register to the RAM bank number plus 2 (ie., 2 or 3) instead of the load/verify switch (0/1).
```

Reply:
Oops, resurrecting a six months old PR; clarified docs